ironfish-cli/src/commands/wallet/encrypt.ts

```diff
@@ -0,0 +1,53 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+import { RpcRequestError } from '@ironfish/sdk'
+import { Flags } from '@oclif/core'
+import { IronfishCommand } from '../../command'
+import { RemoteFlags } from '../../flags'
+import { inputPrompt } from '../../ui'
+
+export class EncryptCommand extends IronfishCommand {
+  static hidden = true
+
+  static description = 'Encrypt accounts in the wallet'
```

Comment:
```suggestion
  static description = 'encrypt accounts in the wallet'
```

part of the cli changes is that these are lower-cased now